### DISCUSSION TITLE
Enable browser logging to get console.log

### DISCUSSION THREAD

**Discussion by szabolcslevay** [ElSnoMan/pyleniumio#238] [Q&A] [answered]
How can I set up the below capability/option in pylenium.json?

```
# enable browser logging
d = DesiredCapabilities.CHROME
d['goog:loggingPrefs'] = { 'performance':'ALL' }
driver = webdriver.Chrome(desired_capabilities=d)
```

or even better, as `ChromeOptions`

```
options = webdriver.ChromeOptions()
options.set_capability("goog:loggingPrefs", {"performance": "ALL"})
driver = webdriver.Chrome(options=options)
```

**Comment by ElSnoMan** [comment, marked as answer]
`DesiredCapabilities` are deprecated, so we use the `Options` class of the specified browser. In the end, you would only need to update the dictionary via the CLI or `pylenium.json`.

For example:

```json
{
  "driver": {
    "browser": "chrome",
    "remote_url": "",
    "wait_time": 10,
    "page_load_wait_time": 0,
    "options": [],
    "capabilities": {
      "goog:loggingPrefs": { "performance": "ALL" }
    },
    ...
}
```

**Reply by ElSnoMan** [reply]
#241 complete. As of Pylenium `v1.14.3`, you can pass in capabilities to the `ChromeDriver` as shown above in the `pylenium.json` example.

> 💡 `capabilities` works for _all_ browsers, but there are browser-specific options and capabilities to be aware of

**Comment by szabolcs-levay** [comment]
That's awesome, @ElSnoMan! Already tested and it's working!
Keep up the good job!
